core/pom.xml

```diff
@@ -614,6 +614,12 @@ THE SOFTWARE.
       <artifactId>jzlib</artifactId>
       <version>1.1.3-kohsuke-1</version>
     </dependency>
+    <!-- Should be removed after splitting maven-builder-plugin see JENKINS-26462 -->
+    <dependency>
```

Comment:
I think you meant to put this in `war/pom.xml`, no? Or maybe that trick does not even work any more in 2.x.